kernel/kernel-api/src/main/java/io/delta/kernel/internal/TableImpl.java

```diff
@@ -146,7 +150,7 @@ protected Path getLogPath() {
    *     range
    * @throws KernelException if provided an invalid version range
    */
-  public CloseableIterator<ColumnarBatch> getChangesByVersion(
+  private CloseableIterator<ColumnarBatch> getRawChanges(
```

Comment:
Common convention for private methods to come after public methods, can we move this below?